packages/solid/src/generators/init/lib/add-dependencies.ts

```diff
@@ -0,0 +1,11 @@
+import { addDependenciesToPackageJson, Tree } from '@nrwl/devkit';
+
+export function updateDependencies(tree: Tree) {
+  const devDependencies = {
+    'solid-jest': '^0.2.0',
+    solid: '^1.2.5',
```

Comment:
Shouldn't this be 'solid-js'? If I call the generator it's asking because the version is not found. Seems like wrong dependency

Reply:
Yup, correct, should be `solid-js`